src/frontend/components/property-type/base-property-component.tsx

```diff
@@ -69,6 +69,7 @@ const BasePropertyComponent: React.FC<BasePropertyComponentProps> = (props) => {
   }), [baseProperty])
 
   const testId = `property-${where}-${property.path}`
+  const contentTag = `${resource.id}-${where}-${property.path}`
```

Comment:
`getActionElementCss`? `where` is just an action name

Reply:
done